app/models/goldencobra/article_url.rb

```diff
@@ -0,0 +1,50 @@
+module Goldencobra
+  class ArticleUrl < ActiveRecord::Base
+    belongs_to :article, class_name: Goldencobra::Article
+    web_url :url
+
+    validates :article_id, presence: true
+    validates :url, presence: true, uniqueness: { case_sensitive: false }
+
+    def self.setup(article_id)
+      article = Goldencobra::Article.find_by_id(article_id)
+
+      # wenn der Artikel gelöscht wurde, sollten die has_many :urls im goldencobra Article
+      # ebenfalls gelöscht sein. Somit ist hier nichts mehr zu tun
+      return true unless article
+
+      # are multiple domains in use?
+      if Goldencobra::Domain.any?
+        setup_domain_urls(article)
```

Comment:
i need more than the id of the article in setup_domain_urls() so i would have another call to db to load the article agin